helm-chart/splunk-kubernetes-logging/templates/_helpers.tpl

```diff
@@ -123,15 +123,34 @@ def extract_container_info:
 .record | extract_container_info | .sourcetype = (find_sourcetype(.pod; .container_name) // "kube:container:\(.container_name)")
 {{- end -}}
 
+
+{{- define "splunk-kubernetes-logging.set_sourcetype" -}}
```

Comment:
I think this function is not being used. Is it redundant or good to have in the codebase.